commands/fncmd.go

```diff
@@ -43,10 +43,10 @@ func GetFnCommand(name string) *cobra.Command {
 		},
 	}
 
-	run := commands.RunCommand(name)
-	run.Short = fndocs.RunShort
-	run.Long = fndocs.RunShort + "\n" + fndocs.RunLong
-	run.Example = fndocs.RunExamples
+	eval := commands.EvalCommand(name)
+	eval.Short = fndocs.RunShort
+	eval.Long = fndocs.RunShort + "\n" + fndocs.RunLong
+	eval.Example = fndocs.RunExamples
```

Comment:
Where is the examples and command reference being updated?

See comment below.